packages/application-shell/src/components/locale-switcher/locale-switcher.tsx

```diff
@@ -59,14 +62,53 @@ const CustomMenuList = (props: MenuListProps) => {
   return <components.MenuList {...props}>{props.children}</components.MenuList>;
 };
 
+export const CustomGroupHeading = (
+  props: GroupHeadingProps & { setIsOpen: (value: boolean) => void }
+) => {
+  const { setIsOpen, ...groupProps } = props;
+  return (
+    <>
+      <components.GroupHeading
+        {...groupProps}
+        css={css`
+          display: flex;
+          gap: ${designTokens.spacing10};
+        `}
+      >
+        {groupProps.children}
+        <IconButton
+          icon={<InformationIcon />}
+          label="Locales info"
+          size="small"
+          onClick={() => setIsOpen(true)}
+        />
+      </components.GroupHeading>
+    </>
+  );
+};
+CustomGroupHeading.displayName = 'CustomGroupHeading';
+
 const LocaleSwitcher = (props: Props) => {
   const { setProjectDataLocale } = props;
+  const [isOpen, setIsOpen] = useState(false);
+
   const handleSelection = useCallback(
     (event) => {
       setProjectDataLocale(event.target.value);
     },
     [setProjectDataLocale]
   );
+
+  const localeOptions = [
+    {
+      label: 'Locales',
```

Comment:
I think we should also translate this word.

Reply:
Added the locale label translations here - [909b857](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/909b857ce5ed1ca4b02d078c1679b74313361877).
Also added the project label translation - [06a07d1](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/06a07d1b79d0762b2bacbc893fe48e2a55de3486).